community-operators/cluster-logging/clusterlogging.v0.0.1.clusterserviceversion.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   name: clusterlogging.v0.0.1
   namespace: placeholder
   annotations:
-    categories: "openshift optional, logging, aggregated, elasticsearch, kibana, fluentd, efk"
+    categories: "Optional OpenShift, Logging & Tracing"
```

Comment:
Should this be `OpenShift Optional`?

Reply:
Yes, whoops I decided to change this order half way through :)